src/Microsoft.TestPlatform.ObjectModel/TestResult.cs

```diff
@@ -40,6 +41,8 @@ public TestResult(TestCase testCase)
             this.TestCase = testCase;
             this.Messages = new Collection<TestResultMessage>();
             this.Attachments = new Collection<AttachmentSet>();
+            this.StartTime = DateTime.Now;
```

Comment:
We should use a better default. Is `null` not okay for serialization/deserialization?